### PR TITLE
Add pause and resume icons to timeline and debugger.

### PR DESCRIPTION
**Pull request by kenzieschmoll**
screen shot 2019-01-30 at 12 26 13 pm
screen shot 2019-01-30 at 12 26 21 pm

### CODE REVIEW

File: packages/devtools/lib/src/timeline/timeline.dart [hunk 1]
```diff
@@ -60,17 +61,19 @@ class TimelineScreen extends Screen {
...
       ..primary()
       ..click(_pauseRecording);
 
-    // TODO(kenzie): add resume icon to button and make collapsible.
-    resumeButton = PButton('Resume recording')
+    resumeButton = PButton()
       ..small()
       ..clazz('margin-left')
       ..disabled = true
+      ..add(<CoreElement>[
```
[the remark this answers is not in the document]
Comment (devoncarew) [reply packages/devtools/lib/src/timeline/timeline.dart]: from memory, the octicons icon set doesn't include a pause icon; if changing this, we should also update the debugger page's pause icon (it's currently using a round 'record' icon).

[the remark this answers is not in the document]
Comment (kenzieschmoll) [reply packages/devtools/lib/src/timeline/timeline.dart]: While, I'm here, I'll go ahead and add these to the debugger page as well.

File: packages/devtools/lib/src/ui/primer.dart [hunk 2]
```diff
@@ -68,7 +68,7 @@ class PButton extends CoreElement {
     setAttribute('type', 'button');
     setAttribute('title', title ?? text);
     if (icon != null) {
-      add(createIconElement(icon));
+      add(createIconElement(_icon = icon));
```
[the remark this answers is not in the document]
Comment (kenzieschmoll) [reply packages/devtools/lib/src/ui/primer.dart]: done.

File: packages/devtools/lib/src/timeline/timeline.dart [hunk 3]
```diff
@@ -165,23 +166,29 @@ class TimelineScreen extends Screen {
...
-    pauseButton.disabled = false;
-    resumeButton.disabled = true;
-
+    _updateButtons(paused: false);
     _paused = false;
-
     _updateListeningState();
   }
 
+  void _updateButtons({@required paused}) {
```
[the remark this answers is not in the document]
Comment (kenzieschmoll) [reply packages/devtools/lib/src/timeline/timeline.dart]: done.

**Pull request merged**